src/utils/Helpers.ts

```diff
@@ -0,0 +1,9 @@
+export class Helpers {
+    static useNestedForLoop(callBack: (i: number, j: number) => void, size = 8): void {
```

Comment:
można zmienić nazwy parametrów na semantyczne żeby nie myśleć za każdym razem czy 'i' to column czy row